beacon-chain/state/state-native/getters_validator.go

```diff
@@ -278,3 +280,20 @@ func (b *BeaconState) inactivityScoresVal() []uint64 {
 	copy(res, b.inactivityScores)
 	return res
 }
+
+// LastActivatedValidatorIndex provides the validator index that was last known in the state
+func (b *BeaconState) LastActivatedValidatorIndex() (primitives.ValidatorIndex, error) {
```

Comment:
why is this part of the state ? It can just be `core/helpers` function, this method uses nothing internal to the state.

Reply:
It was my suggestion, but you're right it doesn't use any internals.